frontend/packages/console-dynamic-plugin-sdk/src/extensions/yaml-field-types.ts

```diff
@@ -0,0 +1,12 @@
+import { JSONSchema7 } from 'json-schema';
+import { K8sModel } from '../api/common-types';
+
+export type YAMLEditorFieldProps = {
+  value: string;
+  onChange: (value: string) => void;
+  label?: string;
+  model?: K8sModel;
+  schema?: JSONSchema7;
+  showSamples: boolean;
```

Comment:
we can make this optional too (please also update the component's tsdoc)
```suggestion
  showSamples?: boolean;
```